tests/test_template.py

```diff
@@ -0,0 +1,68 @@
+import postgrid
+import os
+import time
+
+
+def setup_module():
+    postgrid.pm_key = os.environ.get('PM_API_KEY')
+
+
+def create_test_template():
+    return postgrid.Template.create(
+        html = '<b>test</b> template, hello {{to.firstName}}!',
+        description = 'Test',
+        metadata={
+            'test': [10, 20]
+        }
+    )
+
+
+def test_create():
+    template = create_test_template()
+
+    assert isinstance(template, postgrid.Template)
+    assert isinstance(template.html, str)
+    assert isinstance(template.description, str)
+    assert template.html == '<b>test</b> template, hello {{to.firstName}}!'
+    assert template.description == 'Test'
+
+
+def test_preview_generated(): 
```

Comment:
This function name does not apply for this particular test. We can just call this `test_retrieve`.

Reply:
Right, pushed the new changes.